packages/azure-themes/src/azure/AzureStyleSettings.ts

```diff
@@ -0,0 +1,138 @@
+import { ITheme } from 'office-ui-fabric-react';
+import { CheckboxStyles } from './styles/Checkbox.styles';
+import { CheckStyles, DetailsRowStyles, DetailsListStyles } from './styles/DetailsList.styles';
+import { ChoiceGroupOptionStyles } from './styles/ChoiceGroupOptions.styles';
+import { comboBoxStyles } from './styles/ComboBox.styles';
```

Comment:
A little bit inconsistent the naming of these style constants. Some are Pascal case and other ones are Camel case. Due to the fact that sometimes these are style objects and other times these are functions that return style objects in the `fluent-theme` package we decided mainly to use Pascal casing.

Reply:
Agreed.  We'll tighten this up.